Makefile

```diff
@@ -340,7 +340,8 @@ LDFLAGS=-ldflags "-X github.com/projectcalico/calico-containers/calicoctl/comman
 	-X github.com/projectcalico/calico-containers/calicoctl/commands.BUILD_DATE=$(CALICOCTL_BUILD_DATE) \
 	-X github.com/projectcalico/calico-containers/calicoctl/commands.GIT_REVISION=$(CALICOCTL_GIT_REVISION) -s -w"
 
-GLIDE_CONTAINER_NAME?=dockerepo/glide
+GLIDE_CONTAINER_NAME=calico-build/glide
+GLIDE_CONTAINER_MARKER=calico-build/glide_container.created
```

Comment:
At least in the non-Felix repos we usually just have a calico/ prefix for all of our container names .... so calico/glide would be preferable here.